src/main/java/org/folio/circulation/resources/RequestNoticeSender.java

```diff
@@ -190,6 +195,7 @@ private CompletableFuture<Result<Request>> fetchMissingLocationDetails(Request r
     Item item = request.getItem();
 
     if (item == null || item.isNotFound() || item.getLocation() == null) {
+      log.info("fetchMissingLocationDetails:: location cannot be fetched, item: {}", () -> item);
```

Comment:
do we really need to log the whole item here in info? maybe an item ID will suffice?